hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneManagerVersion.java

```diff
@@ -54,6 +54,9 @@ public enum OzoneManagerVersion implements ComponentVersion {
 
   S3_LIST_MULTIPART_UPLOADS_PAGINATION(11,
       "OzoneManager version that supports S3 list multipart uploads API with pagination"),
+
+  SNAPSHOT_READ_FROM_NON_LEADER(12, "OzoneManager version that supports " +
+      "snapshot read operations from non-leader"),
```

Comment:
nit: can you change non-leader to follower everywhere?

Reply:
I intentionally use "non-leader" term so that it also includes future OM listener. See https://issues.apache.org/jira/browse/HDDS-11523.